templates/default/core-site.xml.erb

```diff
@@ -201,6 +201,11 @@
     <value><%= @defaultFS %></value>
   </property>
 
+    <property>
+    <name>fs.defaultFS.alternativeScheme</name>
+    <value><%= @alternativeScheme %></value>
```

Comment:
The `@alternativeScheme` works only if you pass it as a variable in the template resource in the default.rb recipe - I think in this case we can just hard code `hdfs` here.

Reply:
Yea, I missed the default.rb part. I guess we can hardcode for now and change in the future if it become needed.

Reply:
especially as it is hardcoded in the default.rb :D